style.css

```diff
@@ -399,3 +407,147 @@ form > button:hover {
   align-items: center;
   height: 70px;
 }
+
+@media screen and (min-width: 780px) {
+  body,
```

Comment:
- [ ] The media queries for the Desktop breakpoint (the value for the min-width property should be in the range of 992px - 1100px) as this is the standard breakpoint for desktop.